### ISSUE TITLE
Жизни игрока и оружие

### ISSUE THREAD

**Issue by 5LICK** [ziggi/space-fly#1]
Как насчет добавить жизни для игрока, (3 попадания в него) или healthbar и пару видов оружия?

**Comment by ziggi** [comment]
Согласен. Ещё можно добавлять жизнь или хп за каждые 50 убийств, например. Как-нибудь займусь :)

**Comment by 5LICK** [comment]
Отлично, я думаю тоже что то сделать из этого, потом кину пул реквест что получилось
